src/main/flatbuffers/rlbot.fbs

```diff
@@ -158,6 +165,31 @@ table GameTickPacket {
 
 root_type GameTickPacket;
 
+/// The state of a rigid body in RL's physics engine.
+table RBState {
+  frame:int;
+  quaternion:Quaternion;
```

Comment:
I'm thinking about changing this to rotation:Quaternion. Any objections?

Reply:
If you're going to do that, can you move it below location as well? e.g. the field order will become:

```
table RBState {
  frame:int;
  location:Vector3;
  rotation:Quaternion;
  velocity:Vector3;
  angularVelocity:Vector3;
}
```

(Because we shouldn't mix conventions)

Reply:
Wilco.